fhevm/tee_interpreter.go

```diff
@@ -13,12 +13,12 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
-func doOperationGeneric[T any](
-	environment EVMEnvironment, 
-	caller common.Address, 
-	input []byte, 
-	runSpan trace.Span, 
-	operator func(uint64, uint64) T,
+func doOperationGeneric(
+	environment EVMEnvironment,
+	caller common.Address,
+	input []byte,
+	runSpan trace.Span,
+	operator func(a, b any) any,
```

Comment:
@lovenoble I think we should hardcode to:

```suggestion
	operator func(a, b *big.Int) *big.Int,
```

This way, we don't need to do all the `switch` cases

Reply:
So we should keep it like this for now?

Reply:
Yes